AIDevGallery/Samples/Open Source Models/Embeddings/SemanticSearch.xaml.cs

```diff
@@ -129,6 +129,8 @@ public void Search(string sourceText, string searchText)
             return;
         }
 
+        SendSampleInteractedEvent("Search"); // <exclude-line>
```

Comment:
We should auto exclude lines that call SendSampleInteractedEvent to avoid having to add exclude-line

Reply:
I didn't want to add yet another magical exclude pattern.
Most of these are in blocks that are already being removed, like `NarratorHelper.Announce(...) // <exclude-line>`

Reply:
these should never make it to the exported samples - lets just make sure they don't end up in sample code

Reply:
Done.